packages/lesswrong/components/tagging/TagSubforumPage.tsx

```diff
@@ -40,18 +76,34 @@ export const TagSubforumPage = ({ classes, user }: { classes: ClassesType; user:
     return <Error404 />;
   }
 
-  if ((error && !isMissingDocumentError(error))) {
+  if (error && !isMissingDocumentError(error)) {
     return (
       <SingleColumnSection>
         <Typography variant="body1">{error.message}</Typography>
       </SingleColumnSection>
     );
   }
 
+  const welcomeBoxComponent = (
+    <div className={classes.welcomeBoxPadding}>
+      <div className={classes.welcomeBox}>
+        <ContentStyles contentType="comment">
+          <ContentItemBody
+            dangerouslySetInnerHTML={{ __html: tag.subforumWelcomeText?.html || "" }}
+            description={`${tag.name} subforum`}
+          />
+        </ContentStyles>
+      </div>
+    </div>
+  );
+
   return (
     <div className={classes.root}>
+      <SingleColumnSection className={classNames(classes.columnSection, classes.stickToBottom)}>
+        {welcomeBoxComponent}
+      </SingleColumnSection>
```

Comment:
This should be excluded if there's no welcome message. Also I don't think you need to use `SingleColumnSection` here?

Reply:
@Will-Howard FYI the box is still visible even with no message (tested on staging):

<img width="1439" alt="Screen Shot 2022-09-14 at 2 32 13 PM" src="https://user-images.githubusercontent.com/9057804/190235116-289f3229-b015-4300-adf2-7979b309d52e.png">